packages/types/surveys/types.ts

```diff
@@ -621,31 +621,35 @@ export const ZSurveyMatrixQuestion = ZSurveyQuestionBase.extend({
 
 export type TSurveyMatrixQuestion = z.infer<typeof ZSurveyMatrixQuestion>;
 
-const ZSurveyShowRequiredToggle = z.object({
+const ZInputFieldConfig = z.object({
```

Comment:
the name `ZInputFieldConfig` seems very generic, can we name it to `ZToggleInputConfig`?